api-docs/cloud/ref.yml

```diff
@@ -11039,35 +11038,135 @@ paths:
         - System information endpoints
   /api/v2/scripts:
     get:
+      description: >
+        Retrieves a list of
+        [scripts](/influxdb/cloud/api-guide/api-invokable-scripts/).
+
+
+        #### Limitations
+
+
+        - Paging with an `offset` greater than the number of records will result
+        in
+
+        an empty response--for example:
```

Comment:
Is the extra whitespace here on purpose?

Reply:
I think 

> Is the extra whitespace here on purpose?

I think it gets added by openapi-cli.